app/assets/javascripts/hosts.js

```diff
@@ -56,4 +56,31 @@ $(document).on('ContentLoad', function() {
       wait_msg.modal('show');
     }
   });
+
+  triggerClickToActivateTab();
 });
+
+var triggerClickToActivateTab = function() {
+  /**
+   * Takes the hash part from a url like "#templates" in this example:
+   * 'https:/localhost:3000/{host-uuid}#templates'
+   */
+  var urlHash = window.location.hash;
+  if (!urlHash) return;
+
+  var intervalCount = 0;
+  var intervalID = window.setInterval(function() {
```

Comment:
instead of an interval I tried to use some of jQuery subscribers,
though on `ContentLoad` is happening before the tabs element exist,
and `window onLoad` happens too late, so the "click" is visible to the user and the transition between the tabs is weird, like an attacker started to play with those tabs :rofl: 

IMO with the interval it looks very smooth.
The interval will stop after 4 runs (1 second) or after finding the correct tab.